code/game/gamemodes/crash.dm

```diff
@@ -0,0 +1,154 @@
+/datum/game_mode/crash
+	name = "Crash"
+	config_tag = "Crash"
+	required_players = 0
+	flags_round_type = MODE_INFESTATION|MODE_FOG_ACTIVATED
+	flags_landmarks = MODE_LANDMARK_SPAWN_XENO_TUNNELS|MODE_LANDMARK_SPAWN_MAP_ITEM
+
+	// Round start conditions
+	var/xeno_required_num = 1
+	var/xeno_starting_num = 0
+	var/list/xenomorphs = list()
```

Comment:
Could save on performance and this to init

Reply:
Do you mean specifically the init of the list?

Reply:
Yes, as for why check the contribution guidelines